lib/utilities/global-npm-version.js

```diff
@@ -0,0 +1,9 @@
+// Get the global npm version. You cannot require global npm modules, including
+// global npm itself, but you can still *run* global npm.
+const spawnSync = require('child_process').spawnSync;
```

Comment:
is there a point to telling us what the installed version of npm is? It has nothing to do with ember-cli. i would just update the output to say bundled-npm, if people want to now their version of npm they can run npm version